benchmarks/benchmarks/stats.py

```diff
@@ -19,4 +18,84 @@ def setup(self, *args):
     def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
-            anderson_ksamp(self.rand)
+            stats.anderson_ksamp(self.rand)
+
+
+class CorrelationFunctions(Benchmark):
+    param_names = ['alternative']
+    params = [
+        ['two-sided', 'less', 'greater']
+    ]
+
+    def setup(self, mode):
+        a = np.random.rand(2,2)*10
+        self.a = a
+
+    def time_fisher_exact(self, alternative):
+        if alternative == 'two-sided':
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='two-sided')
+        elif alternative == 'less':
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='less')
+        else:
+            oddsratio, pvalue = stats.fisher_exact(self.a, alternative='greater')
```

Comment:
Oops, my bad. As they were params and initially I wasn't sure how they work , wrote separate blocks. Changed now. Thanks.